confidence/io.py

```diff
@@ -302,3 +302,25 @@ def generate_sources() -> typing.Iterable[typing.Mapping[str, typing.Any]]:
                 yield loadf(candidate, default=NotConfigured)
 
     return Configuration(*generate_sources(), missing=missing)
+
+
+def dumpf(configuration: Configuration, fname: typing.Union[str, PathLike], encoding: str = 'utf-8') -> None:
+    """
+    Serialize the configuration in *configuration* to a YAML-formatted file.
+
+    :param configuration: the `.Configuration` object to dump
+    :param fname: name or path of the file to write to
+    :param encoding: encoding to use
+    """
+    with open(fname, 'w', encoding=encoding) as out_file:
+        out_file.write(dumps(configuration))
+
+
+def dumps(configuration: Configuration) -> str:
+    """
+    Serialize the configuration in *configuration* as a YAML-formatted string.
+
+    :param configuration: the `.Configuration` object to dump
+    :return: *configuration*, serialized as a `str` in YAML format
+    """
+    return yaml.safe_dump(configuration._source, default_flow_style=False)
```

Comment:
Maybe add a line comment on why the default flow style isn't good?

Reply:
Could be nice. I wouldn't know what's bad about it.

Reply:
FXIED.